linux/LinuxProcessList.c

```diff
@@ -533,6 +533,33 @@ static void LinuxProcessList_readCGroupFile(LinuxProcess* process, const char* d
       if (!ok) break;
       char* group = strchr(buffer, ':');
       if (!group) break;
+
+      if (!process->lxc) {
```

Comment:
Does the LXC cgroup never change for a running process? (Cause it's only computed once)

Reply:
Technically it could (changed process namespace), but practically it doesn't.